src/views/blocks/rsvp/form/details.php

```diff
@@ -12,10 +12,11 @@
  *
  * @since   4.9
  * @since   4.11.5 Corrected the template override instructions in template comments.
+ * @since   TBD Add $post_id to filter for hiding opt-outs.
  *
- * @version 4.11.5
+ * @version TBD
  */
 
 $this->template( 'blocks/rsvp/form/name', [ 'ticket' => $ticket ] );
 $this->template( 'blocks/rsvp/form/email', [ 'ticket' => $ticket ] );
-$this->template( 'blocks/rsvp/form/opt-out', [ 'ticket' => $ticket ] );
\ No newline at end of file
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: ba5b5b70077a69a290dfd281da6040d1</sub>